Sources/AppStoreConnectCLI/Arguments/BuildLocalizationInputArguments.swift

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import FileSystem
+import Foundation
+
+struct BuildLocalizationInputArguments: ParsableArguments {
+    @Argument(help: "The locale information of the build localization resource. eg. (en-AU)")
+    var locale: String
+
+    @Option(
+        parsing: SingleValueParsingStrategy.scanningForValue,
+        help: """
+            A field that describes changes and additions to a build and indicates features you would like your users to test.
+            Please put the text inside an ''.
+            eg. ('Hello There')
+        """
+    )
+    var whatsNew: String?
+
+    @Option(help: "File path to a txt file that contains whats new text. (eg. whatsNew.txt)")
+    var path: String?
+}
+
+protocol CreateUpdateBuildLocalizationCommand {
+    var localization: BuildLocalizationInputArguments { get }
+}
+
+extension CreateUpdateBuildLocalizationCommand {
+
+    func validateWhatsNewInput() throws {
+        if localization.whatsNew != nil && localization.path != nil {
+            throw ValidationError("Please provide either a file path or whats new text in terminal.")
+        }
+    }
+
+    var whatsNew: String {
+        let whatsNew: String
+
+        if let whatsNewFromOption = localization.whatsNew {
+            whatsNew = whatsNewFromOption
+        } else if let filePath = localization.path {
+            guard let whatsNewString = try? String(contentsOfFile: filePath) else {
+                fatalError("Unable to read text from file path")
```

Comment:
Should we just throw the error and make getting what's new a function?

Reply:
Were you going to make `readWhatNew` a throwing function?

Reply:
@adamjcampbell Yes, I just changed the function to a throwing function. `throw Error.invalidFilePath(filePath)`